docs/changelog/2719.feature.rst

```diff
@@ -0,0 +1,3 @@
+Modified handling of NO_COLOR environment variable, consistent with de facto conventions (see https://no-color.org);
```

Comment:
Can you make this link restructuredtext link please

Reply:
Also quote with double backtick the env var name

Reply:
Updated - sorry, I don't use .rst much anymore